js-src/src/app/map.component.js

```diff
@@ -125,21 +123,23 @@ let MapComponent = Component({
                                 isOpen: 'true',
                                 details: currentCity.name
                             });
-
-                            if (self.currentRound <= self.markers.length) {
-                                self.startCountdown(1);//Initialize Break timer
-                            } else {
-                                //Game ended, navigate to highscore page
-                                this.gameEnded = true;
-                                self.router.navigate(['/highscores', self.game.id, self.player.id])
-                            }
+							
                             self.guessService.getAllGuesses(self.game.id).subscribe(function (guesses) {
                                 for (var i = 0; i < guesses.length; i++) {
                                     if ((guesses[i].player.id != self.player.id) && (guesses[i].city.id == currentCity.id)) {
                                         var pin2 = self.getPinImage(self, guesses[i].player.id);
                                         self.playerMarkers.push({lat: guesses[i].lat, lng: guesses[i].long, img: pin2});
                                     }
                                 }
+								if (self.currentRound < 2) {//Here we have got 3 rounds each game
```

Comment:
I have tried console.log(self.markers.length), it is always 1, it will be cleared each round(You can see it in handleCountdown(1)). So if we have 5 rounds, it will go to else statement. Is this right?